client/package.json

```diff
@@ -42,7 +42,7 @@
     "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-react-refresh": "^0.4.5",
     "typescript": "^5.2.2",
-    "vite": "^5.0.12",
+    "vite": "^4.4.0",
```

Comment:
Could we provide the reason why the version has been downgraded?

Reply:
Sorry, not really sure, but I found this tab left open by @iboshkov 
https://stackoverflow.com/questions/77583341/cannot-find-module-rollup-rollup-win32-x64-msvc-npm-has-a-bug-related-to-optio